[2025-04-26T18:55:07.960+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: email_sender.extract_data manual__2025-04-26T18:55:06.248247+00:00 [queued]>
[2025-04-26T18:55:07.963+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: email_sender.extract_data manual__2025-04-26T18:55:06.248247+00:00 [queued]>
[2025-04-26T18:55:07.964+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-26T18:55:07.968+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_data> on 2025-04-26 18:55:06.248247+00:00
[2025-04-26T18:55:07.970+0000] {standard_task_runner.py:60} INFO - Started process 117 to run task
[2025-04-26T18:55:07.972+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'email_sender', 'extract_data', 'manual__2025-04-26T18:55:06.248247+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/email_sender_dag.py', '--cfg-path', '/tmp/tmprshdujm8']
[2025-04-26T18:55:07.973+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extract_data
[2025-04-26T18:55:07.994+0000] {task_command.py:423} INFO - Running <TaskInstance: email_sender.extract_data manual__2025-04-26T18:55:06.248247+00:00 [running]> on host ebc446b713fb
[2025-04-26T18:55:08.020+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='email_sender' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-26T18:55:06.248247+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-26T18:55:06.248247+00:00'
[2025-04-26T18:55:08.024+0000] {base.py:83} INFO - Using connection ID 'postgresql_padrao' for task execution.
[2025-04-26T18:55:08.040+0000] {warnings.py:109} WARNING - /opt/***/dags/email_sender_dag.py:59: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(sql_query, conn)

[2025-04-26T18:55:08.053+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-26T18:55:08.056+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=email_sender, task_id=extract_data, execution_date=20250426T185506, start_date=20250426T185507, end_date=20250426T185508
[2025-04-26T18:55:08.070+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-26T18:55:08.080+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
